hermeznetwork/bridge-ui
# Move PriceOracle to Uniswap instead of 1Inch

### What does this PR does?

This PR changes the implementation of the `PriceOracle` to use Uniswap instead 1Inch. This is important for us as 1Inch is contracts are not deployed in Rinkeby, but Uniswap contracts are supported on all Ethereum testnets.

### How to test?

You can add this code to the `Home` view:

```ts
const { getTokenPrice } = usePriceOracleContext();

useEffect(() => {
  getTokenPrice("0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0")
    .then((res) => console.log(formatUnits(res, 6)))
    .catch(console.error);
}, [getTokenPrice]);
```

It should return the current price of the MATIC token in the Ethereum mainnet.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
